### PR TITLE
Added documentation guide

### PR DESCRIPTION
**Pull request by parulagg27**
**Fixes issue:** #3261 

**Changes:** Added template for writing notes for each algorithm.

### CODE REVIEW

File: guides/documentation_guide.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Writing notes for each algorithm
+When creating the notes for algorithms of various categories, please use this template:
+
+```
```
Comment (arnavb) [inline guides/documentation_guide.md]: Write `Markdown` after the three ticks so proper syntax highlighting is visible.

File: guides/documentation_guide.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# Algorithm Name
+Description
+
+## Explanation
+Pictorial diagram to explain the algorithm 
+
+> Image credits: (name of the site/organisation from where the image was referenced)
+
+## Algorithm
+Algorithm in words
```
Comment (arnavb) [inline guides/documentation_guide.md]: How about, "Algorithm as a numbered list"?

File: guides/documentation_guide.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+### PseudoCode
```
Comment (arnavb) [inline guides/documentation_guide.md]: If you go with my previous suggestion, you may want to omit this header. (Suggestion)

Comment (parulagg27) [reply guides/documentation_guide.md]: Yeah, agree. Will do that!

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**